app/views/finders/show.atom.builder

```diff
@@ -8,7 +8,7 @@ atom_feed do |feed|
   end
 
   @feed.entries.each do |result|
-    feed.entry(result, id: result.path, url: absolute_url_for(result.path), updated: DateTime.parse(result.last_update)) do |entry|
+    feed.entry(result, id: "tag:#{Plek.current.website_root.gsub('http://www.', '')},#{feed.instance_variable_get(:@feed_options)[:schema_date]}:#{result.path}", url: absolute_url_for(result.path), updated: DateTime.parse(result.last_update)) do |entry|
```

Comment:
This line is now really long and the tag logic seems complex enough to belong in a presenter.